plugins/importexport/crossref/README

```diff
@@ -1,14 +1,16 @@
 ================================
 === OJS CrossRef Export Plugin
-=== Version: 1.1
-=== Release date: June 8, 2006
-=== Author: Juan Pablo Alperin <juan@geeky.net>
+=== Version: 2.1
+=== Author: Bozana Bokan <bozana.bokan@posteo.net>
+=== Author: Juan Pablo Alperin <juan@alperin.ca>
+=== Author: James MacGregor <jmacgreg@gmail.com>
+=== Last update: July 2016
```

Comment:
Better not to include "last update", since that info is available via github and we're likely not to update this consistently.